tests/fs/test_trackingdelegate.c

```diff
@@ -123,8 +123,6 @@ int main() {
   read(fd, output, 100);
   printf("read returned %s\n", output);
   close(fd);
-  fd = open("/renamed.txt", O_CREAT);
-  close(fd);
```

Comment:
Hmm, this looks like a valid test, so there might be an underlying issue we need to fix, unless I'm missing something?

Reply:
Yes that certainly looks odd.  I wonder if we should land this as is to green the asan builder?  I guess there is no need since that is just an FYI builder and doesn't actually hold us up.

Reply:
in other works I think its probably work investigating rather than landing this in a hurry.